### ISSUE TITLE
Alex may suggest donating to the activist, then get mad when you do

### ISSUE THREAD

**Issue by scgtrp** [i7/counterfeit-monkey#194]
I got this silly sequence while talking to the activist in University Oval:
```
I'm inclined to donate.

>donate

Oh, you're not seriously going to -- oh, god, you are. You know, this is never going to work. There are sound scientific reasons why -- oh FINE.
```

"I'm inclined to" is randomly chosen text that happened to be probably the worst option here; it looks fine if it chooses "We could" instead.

I made an attempt at fixing it myself for fun, but decided not to submit this as a real pull request because I have no idea what I'm doing and there is probably a much better way to do this. Feel free to use or discard the following as desired.
<details>

```diff
+A quip can be Alex-approved. A quip is usually Alex-approved.
+
+The saved quip-suggestion-phrase is text which varies. The saved quip-suggestion-phrase is "".
+Before listing plausible quips when any plausible quip is not Alex-approved:
+       now the saved quip-suggestion-phrase is the quip-suggestion-phrase;
+       now the quip-suggestion-phrase is "[We] [could] ".
+After listing plausible quips when the saved quip-suggestion-phrase is not "":
+       now the quip-suggestion-phrase is the saved quip-suggestion-phrase;
+       now the saved quip-suggestion-phrase is "".
+

[...]

+       donate is not Alex-approved.
```
</details>

**Comment by angstsmurf** [comment]
Thanks a lot for this! That is a good point, and a good fix.

After spending so much time on improving the performance of this game, I'm a little paranoid of making changes that will slow it down, which adding a new global property (Alex-approved) would, even if it would be imperceptible, so I prefer to just check whether the "donate" quip is plausible.

If we find more examples of this, it might be better to do it your way and use a general mechanism.

Should be fixed by 0235ee6.

**Comment by scgtrp** [comment]
Yep, that would be the much better way I didn't see. Thanks!
